sci-biology/squizz/squizz-0.99b.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="squizz is a sequence format file checker, but it has some
+conversion capabilities too."
+HOMEPAGE=""
+SRC_URI="ftp://ftp.pasteur.fr/pub/gensoft/projects/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_configure(){
+    econf
+}
+
+src_compile() {
+    emake 
+}
+
+src_test(){
+    einfo ">>> Test phase [check]: ${CATEGORY}/${PF}"
+    make  -j1 check || die "test failed"
+}
+
+src_install() {
+    emake DESTDIR="${D}" install 
+}
```

Comment:
So if I understand I specify only the functions wich must differ that the defautl one
the pipeline is always the same 
 pkg_setup, src_unpack, src_prepare, src_compile, src_test (optional, FEATURES="test"), src_install, pkg_preinst, pkg_postinst
if the function is not define, it take the defaut function corresponding to the step and matching the EAPI?